lib/paratii.core.users.js

```diff
@@ -11,5 +11,49 @@ export class ParatiiCoreUsers {
     }
     let options = dopts(config, defaults, {allowUnknown: true})
     this.config = options
+    this.paratii = this.config.paratii
+  }
+
+  async create (options) {
+    let keysForBlockchain = ['id', 'name', 'email']
+    let optionsKeys = Object.keys(options)
+    let optionsBlockchain = {}
+    let optionsIpfs = {}
+    optionsKeys.forEach(function (key) {
+      if (keysForBlockchain.includes(key)) {
+        optionsBlockchain[key] = options[key]
+      } else {
+        optionsIpfs[key] = options[key]
+      }
+    })
+    let hash = await this.paratii.ipfs.addJSON(optionsIpfs)
+    optionsBlockchain['ipfsHash'] = hash
+    return this.paratii.eth.users.create(optionsBlockchain)
+  }
+
+  get (id) {
+    return this.paratii.db.users.get(id)
+  }
+
+  async update (userId, options) {
+    let defaults = {
+      name: null, // must be a string, optional
+      email: null, // must be a string, optional
+      ipfsHash: null // must be a string, optional, default is ''
+    }
```

Comment:
So the "options" for `create` and `update` should have the same allowed values (expect for the `id`, that is optional for the `create` but not valid for the `update`). The `ipfsHash` is not one of them.

Reply:
Also, for consistency, we should rename `ipfsHash` in the paratii.eth.users.create` to `ipfsData` (which is the name of the method used on the contracts).